src/nylo/parser.py

```diff
@@ -1,3 +1,27 @@
+# This file is a part of nylo
```

Comment:
parser.py should not have the copyright. Only file beggining with __ shall have them.